src/components/CommunityCard/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, Button, Heading, Image, Text } from 'grommet';
+
+export const CommunityCard = ({
+  title,
+  link,
+  linkname,
+  description,
+  priority,
+  image,
+  ...rest
+}) => (
+  <Box
```

Comment:
This should be a Grommet Card component with elevation="medium" to match the figma mock

Reply:
I used the card component now with Grid.